guides/security/samesite-protection.md

```diff
@@ -0,0 +1,28 @@
+# SameSite protection
```

Comment:
Maybe add a redirect from the old `guides/plugins/plugins/storefront/use-csrf-protection.md` to the new page, so the link won't break. Did you check for occurrences of the old page link?

Reply:
Yes, I cross-checked with Phpstorms usage functionality but also searched by hand.

I only came across this -> https://developer.shopware.com/docs/guides/hosting/infrastructure/reverse-http-cache#configure-shopware

But this seems unrelated. Maybe @shyim can confirm since he created most of it.

Reply:
http cache should be the only part :D